demo/src/screens/componentScreens/RadioButtonScreen.js

```diff
@@ -39,27 +62,51 @@ export default class RadioButtonScreen extends Component {
   render() {
     return (
       <View flex useSafeArea bg-dark80>
-        <View flex padding-20>
-          <View flex>
-            <View spread row>
-              <RadioGroup value={this.state.color} onValueChange={value => this.setState({color: value})}>
-                <Text marginB-20 text60 dark10>
-                  Select a color
-                </Text>
-                {this.renderRadioButton('orange', 'Orange')}
-                {this.renderRadioButton('purple', 'Purple')}
-                {this.renderRadioButton('green', 'Green')}
-                <Text marginT-10>You chose: {this.state.color}</Text>
-              </RadioGroup>
-              <RadioGroup value={this.state.textSide} onValueChange={value => this.setState({textSide: value})}>
-                <Text marginB-20 text60 dark10>
-                  Select text side
-                </Text>
-                {this.renderRadioButtonWithImageAndText('right', 'Text on right')}
-                {this.renderRadioButtonWithImageAndText('left', 'Text on left', true)}
-                <Text marginT-10>You chose: {this.state.textSide}</Text>
-              </RadioGroup>
-            </View>
+        <View flex>
+          <ScrollView style={{padding: 20}}>
+            <RadioGroup value={this.state.color} onValueChange={value => this.setState({color: value})}>
+              <Text marginB-20 text60 dark10>
+                Select a color
+              </Text>
+              {this.renderRadioButton('orange', 'Orange')}
```

Comment:
since we now support the label, it's easier to render a radiobutton with a label with a one liner
`<RadioButton value={value} label={text} marginB-5/>`
we can clean the example screen a little to remove the `renderRadioButton` function since it doesn't do much now. 

In general, we should keep the example screen clean and readable since that's where we address people to learn how to use our components.

Reply:
This is the section title, the `RadioButton`s have been changed to include the label